codecov.yml

```diff
@@ -20,8 +20,8 @@ ignore:
   - "**/*Modal.swift"
   - "**/*ViewController.swift"
   - "**/*ViewController+Extensions.swift"
-  - "CybridSDK/Components/Trade/TradeViewModel.swift"
-  - "CybridSDK/Components/Crypto Price List/CryptoPriceModel.swift"
+  - "CybridSDK/Components/Trade/View/TradeViewModel.swift"
+  - "CybridSDK/Components/ListPrices/View/ListPricesViewModel.swift"
```

Comment:
Why aren't we testing the model?

Reply:
Is under creation! This is part 1 of 2 !! The second part gonna be test!